WMIAdventure/frontend/src/js/pages/creative/CardsCreator/atoms/LevelChoose/styled-components/Button.js

```diff
@@ -14,13 +14,21 @@ const Button = styled.button`
   font-weight: 300;
   padding: 0;
   border: none;
-  cursor: pointer;
-  background: transparent;
+  cursor: ${
+          ({disabled}) => disabled ? "initial" : "pointer"
+  };
+  background: ${

```

Comment:
Tutaj dałbym background-color, zamiast samego background.
Może zdarzyć się tak, że chcemy niżej coś innego zmienić np. background-image, a to background: X, nadpisuje nam wszystko

Reply:
racja

Reply:
poszedł approve